src/software/Localization/main_cameraLocalizer.cpp

```diff
@@ -236,6 +238,8 @@ int main(int argc, char** argv)
           "[voctree] Maximum matching error (in pixels) allowed for image matching with "
           "geometric verification. If set to 0 it lets the ACRansac select "
           "an optimal value.")
+      ("useFrameBufferMatching", po::bool_switch(&useFrameBufferMatching), 
```

Comment:
yes. So far the size of `_frameBuffer` is hardcoded to 5 in the class constructor of VoctreeLocalization. That needs to be changed